modules/RTC/JitsiLocalTrack.js

```diff
@@ -329,6 +329,10 @@ export default class JitsiLocalTrack extends JitsiTrack {
             return Promise.resolve();
         }
 
+        if (this.disposed) {
+            return Promise.reject(new Error('Track was already disposed'));
```

Comment:
We might have a special JitsiTrackError for that case which is then handled in jitsi-meet

Reply:
JitsiTrackErrors.TRACK_IS_DISPOSED

Reply:
Wait, I'm turning that into a resolve, let's pretend everything is fine, the track is dead anyway :-)

Reply:
yes, I guess that would be fine

Reply:
although I'm not sure about the unmute case for a disposed track...